### DISCUSSION TITLE
How do you delete a Port Mapping?

### DISCUSSION THREAD

**Discussion by zdb999** [caprover/caprover#1127] [Q&A]
Hi,

I would like to enable port mapping to run some migrations in my database, and disable it right after. How should I go about deleting a port mapping? I can change the port number in the UI, but not get rid of it entirely. Also, how does the port mapping work under the hood?

**Comment by githubsaturn** [comment]
You can remove the port mapping by deleting the target and container port, then SAVE and UPDATE
